package.json

```diff
@@ -98,8 +97,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix",
-      "git add"
```

Comment:
after eslint fixes it, the files won't be there unstaged?

Reply:
The newer version automatically runs `git add` now.

Reply:
Oh, that is good!